recipes/brewer2mpl/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set version = "1.4.1" %}
+
+package:
+  name: brewer2mpl
```

Comment:
Appears this is being renamed to `palettable`. Are we needing a feedstock for each or should we just switch to `palettable`?